### PR TITLE
Updates to the README and install steps

### PR DESCRIPTION
**Pull request by jwilander**
I need to follow-up and add a section about configuring HTTPS but figured this was a good start.

### CODE REVIEW

File: docs/examples/simple_aws_azure.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+apiVersion: mattermost.com/v1alpha1
+kind: ClusterInstallation
+metadata:
+  name: mm-example-aws-azure
+spec:
+  ingressName: example.dev.cloud.mattermost.com
+  useServiceLoadBalancer: true
```
Comment (cpanato) [inline docs/examples/simple_aws_azure.yaml]: just add a new line here

File: README.md [hunk 2]
```diff
@@ -1,93 +1,144 @@
 # Mattermost Operator for Kubernetes ![CircleCI branch](https://img.shields.io/circleci/project/github/mattermost/mattermost-operator/master.svg) [![Community Server](https://img.shields.io/badge/Mattermost_Community-cloud_channel-blue.svg)](https://community.mattermost.com/core/channels/cloud)
-Mattermost Operator for Kubernetes is under construction.
 
 ## Status
-
-The Mattermost operator is in alpha, data loss might occur.
+The Mattermost operator is in alpha but being actively developed.
```
Comment (gabrieljackson) [inline README.md]:
```suggestion
The Mattermost operator is in alpha, but is being actively developed.
```
nit:

File: README.md [hunk 3]
```diff
@@ -1,93 +1,144 @@
...
-### Prerequisites
+The Mattermost server source code is available at https://github.com/mattermost/mattermost-server.
 
-- Kubernetes cluster 1.8.0+.
-- `kubectl` configured for the relevant Kubernetes cluster. (https://kubernetes.io/docs/reference/kubectl/overview/)
+## 1. Prerequisites
 
-### Install Operators
+### 1.1 Kubernetes Cluster with `kubectl`
+You must have a running Kubernetes 1.8.0+ cluster. If you do not, then see the [official Kubernetes documentation](https://kubernetes.io/docs/setup/) to get one set up.
```
Comment (gabrieljackson) [inline README.md]: nit: I don't think we have actively tested anything below 1.11.X, so we may want to bump this number up unless you have tried something older.

File: README.md [hunk 4]
```diff
@@ -1,93 +1,144 @@
...
+
+```bash
+$ kubectl delete -f mm_clusterinstallation.yaml
+```
+
+## 3. Developer flow
+To test the operator locally. We recommend [Kind](https://kind.sigs.k8s.io/), however, you can use Minikube or Minishift as well.
+
+### 3.1 Prerequisites
+To developer locally you will need the [Operator SDK](https://github.com/operator-framework/operator-sdk).
```
Comment (gabrieljackson) [inline README.md]:
```suggestion
To develop locally you will need the [Operator SDK](https://github.com/operator-framework/operator-sdk).
```

File: README.md [hunk 5]
```diff
@@ -1,93 +1,144 @@
...
+$ make dep
+$ make install
+```
 
-Developing and testing local changes to the `mattermost-operator` is fairly simple. For that you can deploy Kind and then apply the manifests to deploy the dependencies and to deploy Mattermost operator as well.
+### 3.2 Testing locally
+Developing and testing local changes to Mattermost operator is fairly simple. For that you can deploy Kind and then apply the manifests to deploy the dependencies and the Mattermost operator as well.
 
-You dont need to push the mattermost-operator image to the Docker Hub or any other Registry you can load the image you built using `make build-image` directly to the Kind cluster.
+You dont need to push the mattermost-operator image to DockerHub or any other registry you can load the image, built with `make build-image`, directly to the Kind cluster:
```
Comment (gabrieljackson) [inline README.md]:
```suggestion
You don't need to push the mattermost-operator image to DockerHub or any other registry if testing with kind. You can load the image, built with `make build-image`, directly to the Kind cluster by running the following:
```

**Pull request merged**